Automattic/portkey-android
# changed the name of a frame from page to slide.

Fixes #316  

Builds on top of #388 

## Findings
Change the name of each frame from `page` to `slide`

## Solution
Replaced all string representations that contained page with the word slide. 

## Testing
1. Create a new story. 
2. Add a new frame. 
3. Notice that when you press the menu you see "Delete slide"
4. Notice if you press that option the dialog refers to the frame as "slide"
5. Verify that the string changes in the diff are accurate since all these paths don't have to be tested since these are string changes. 

Menu | Dialog
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/85805530-341f9200-b712-11ea-950a-b4e15db5a289.png" width="320"></kbd> |       <kbd><img src="https://user-images.githubusercontent.com/1509205/85805533-384baf80-b712-11ea-8422-07b450469823.png" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
